index.js

```diff
@@ -5,7 +5,7 @@ const includeDir = path.relative('.', __dirname);
 module.exports = {
   include: `"${__dirname}"`, // deprecated, can be removed as part of 4.0.0
   include_dir: includeDir,
-  gyp: path.join(includeDir, 'node_api.gyp:nothing'),
+  gyp: path.join(includeDir, 'node_addon_api.gyp'),
```

Comment:
Is it all that harmful to leave this as-is and introduce a new key? If there's even a remote chance that it'll break folks, it may be simplest to not reuse this, but introduce a new key, and document what it does.

Reply:
Couldn't folks simply write the following?

```gyp
"<!(node -p \"require('node-addon-api').include_dir\")/node_addon_api.gyp:node-addon-api-except"
```

Then we would need only document that node_addon_api.gyp is the way to get add node-addon-api to one's binding.gyp.

Reply:
I've updated the PR to name the entrypoint as `require('node-addon-api').targets`.